RELEASE_NOTES.md

```diff
@@ -3,12 +3,12 @@
 Changes:
 
  * Allow adding and editing Library creation date
- * Add updating identities with new external names when creating children
- * Add archived Sample Purposes
- * fixed JIRA search for Run-related tickets
- * Allow non-unique indices within a family
+ * Add updating identities with new external names when creating children (DetailedSample)
+ * Add archived Sample Purposes (DetailedSample)
+ * Fixed JIRA search for Run-related tickets
+ * Allow unique dual-index families to have same name for corresponding indices at positions 1 & 2
```

Comment:
Same name was always fine. This change is to allow the same sequence in the same position within the same family